README.md

```diff
@@ -38,6 +38,7 @@ An added advantage of being a completely Kubernetes native solution is that admi
 [🇺🇦](translations/README.ua.md)
 [🇨🇳](translations/README.zh.md)
 [🇫🇷](translations/README.fr.md)
+[:india:](translations/README.gu.md)
```

Comment:
As more indian languages are added, is there way to differentiate between various languages.

Reply:
Yeah, this could be an issue. I'll look at how to differentiate between various languages.